ElasticSearch/legos/elasticsearch_check_health_status/elasticsearch_check_health_status.py

```diff
@@ -11,58 +11,40 @@
 
 
 class InputSchema(BaseModel):
-    host: str = Field(
-        title='Host',
-        description='URL of Elasticsearch server'
-    )
-    port: int = Field(
-        9200,
-        title='Port',
-        description='Port used by Elasticsearch.'
-    )
-    api_key: str = Field(
-        title='API Key',
-        description='API Key for Authentication'
-    )
-
+    pass
 
 def elasticsearch_check_health_status_printer(output):
     if output is None:
         return
     print(output)
 
 
-def elasticsearch_check_health_status(handle,
-                                      host: str,
-                                      port: int,
-                                      api_key: str) -> Dict:
+def elasticsearch_check_health_status(handle) -> Dict:
     """elasticsearch_check_health_status checks the status of an Elasticsearch cluster .
 
             :type handle: object
             :param handle: Object returned from Task Validate
 
-            :type host: str
-            :param host: URL of your Elasticsearch server
-
-            :type port: int
-            :param port: Port used by your Elasticsearch server
-
-            :type api_key: str
-            :param api_key: API Key for authentication of the request
-
             :rtype: Result Dict of result
     """
-    es_path = host + ":"+str(port) + "/_cluster/health?pretty"
-    es_header = "Authorization: ApiKey" + " " + api_key
-    cmd = ["curl", "-k", "-XGET", "-H", es_header, es_path]
-    try:
-        raw_result = subprocess.check_output(cmd, stderr=PIPE, universal_newlines=True, shell=False)
-        new_result = raw_result.replace("\n", "")
-        result_dict = json.loads(new_result)
-        d = {}
-        for key in result_dict:
-            d[key] = result_dict[key]
-        print("The status for " + d["cluster_name"] + " is " + d["status"])
-        return d
-    except subprocess.CalledProcessError as e:
-        return e.output
+    
+    output = handle.web_request("/_cluster/health?pretty",  # Path
+                                "GET",                      # Method
+                                None)                       # Data
+
+    # es_path = host + ":"+str(port) + "/_cluster/health?pretty"
+    # es_header = "Authorization: ApiKey" + " " + api_key
+    # cmd = ["curl", "-k", "-XGET", "-H", es_header, es_path]
+    # try:
+    #     raw_result = subprocess.check_output(cmd, stderr=PIPE, universal_newlines=True, shell=False)
+    #     new_result = raw_result.replace("\n", "")
+    #     result_dict = json.loads(new_result)
+    #     d = {}
+    #     for key in result_dict:
+    #         d[key] = result_dict[key]
+    #     print("The status for " + d["cluster_name"] + " is " + d["status"])
+    #     return d
+    # except subprocess.CalledProcessError as e:
+    #     return e.output
+
+    return output.args
```

Comment:
why are we returning output.args? output of web_request is the response itself, right?

Reply:
<img width="1210" alt="Screenshot 2022-12-20 at 11 40 57 AM" src="https://user-images.githubusercontent.com/87547684/208756254-d7e9a391-8ed4-4fcc-a8f2-cf5996b84b81.png">

the Output is of type `subprocess`  by using .args, we capture both stdout and stderr and return to the calling function.

Reply:
this is not right, so unresolving it.

Reply:
Resovled it. As i am directly using lambda not via a subprocess.